hubblestack/grains/hubble_core.py

```diff
@@ -2559,7 +2533,7 @@ def _hw_data(osdata):
             'serialnumber': 'serial#',
             'productname': 'DeviceDesc',
         }
-        for grain_name, cmd_key in six.iteritems(hwdata):
+        for grain_name, cmd_key in iter(hwdata.items()):
```

Comment:
why iter()? isn't hwdata.items() enough? I think iter might be extra here.

Reply:
In six.py, for Python-3 it was:

def iteritems(d, **kw):
        return iter(d.items(**kw))

So, no change.

Reply:
Right, but isn't for item in dictthing.items() the same thing as iter(dictthing.items()) -- except with an extra iterator around it?

Reply:
You are right. I tried running the test code, and it was the same behavior without extra iter()
I changed it